imports/util/editor-model-factory.js

```diff
@@ -0,0 +1,56 @@
+
+export const initialEditorContent = {
+  streamEditor: {
+    text: 'To begin, please Select a stream or create one.',
+    mode: 'text',
+    name: 'DEFAULT NAME',
+  },
+  localEditor: {
+    text: 'Welcome to Codemo!',
+    mode: 'text',
+  },
+};
+
+export const defaultEditorContent = {
+  streamEditor: {
+    text: '// Welcome to your new Stream!',
+    mode: 'javascript',
+  },
+  localEditor: {
+    text: '// Welcome to Codemo!',
+    mode: 'javascript',
+  },
+};
+
+
+const INIT = { local: true, stream: true };
+
+export function makeEditorModel(type, text, mode) {
+  if (!type) throw new Error('What editor is this model for? Can be \'stream\' or \'local\'');
+  if (INIT.local || INIT.stream) {
```

Comment:
The naming of these properties on `INIT` is confusing. Are they set to `true` when they need to be init-ed? If so, we should name them appropriately, `shouldInit` or `needsInit` or something. Either that or flip the boolean and initialize it to `false`, and then `INIT.local = true` means that it _was_ initiated.